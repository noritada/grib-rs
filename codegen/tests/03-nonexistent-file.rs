use grib_codegen::parameter_codes;

#[parameter_codes(path = "nosuchfile")]
pub enum FooCodes {}

fn main() {}
