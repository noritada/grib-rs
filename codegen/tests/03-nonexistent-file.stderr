error: wrong input file
 --> tests/03-nonexistent-file.rs:3:26
  |
3 | #[parameter_codes(path = "nosuchfile")]
  |                          ^^^^^^^^^^^^
